Makefile

```diff
@@ -155,7 +161,7 @@ $(CONTROLLER_GEN): ## Build controller-gen from tools folder.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.9
 
 $(CONVERSION_GEN): ## Build conversion-gen
-	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) k8s.io/code-generator/cmd/conversion-gen@v0.18.2
+	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) k8s.io/code-generator/cmd/conversion-gen@v0.19.0-beta.2
```

Comment:
do we need the version bump? or is it just "might as well"

Reply:
asking mostly because it's a beta release, maybe we should wait for the actual release

Reply:
It was “might as well”. I left it since it advanced and didn’t change results. I’m happy to revert to the most recent stable.

Reply:
I removed the beta and changed to v0.18.4, which is the latest release.